crypto/conf/conf_sap.c

```diff
@@ -16,10 +16,6 @@
 #include <openssl/asn1.h>
 #include <openssl/engine.h>
 
-#if defined(_WIN32) && !defined(__BORLANDC__)
-# define strdup _strdup
-#endif
-
```

Comment:
Just to ask the question, did you test this on windows?  I'm not going to pretend to know the Windows api (which I certainly hope has the appropriate string functions available, but this will shift the code below to using strcpy instead of the win32 _strdup call.  If theres a configuration in win32 that doesn't have strdup, I'd be curious to know if it has strcpy